pages/community/index.js

```diff
@@ -155,22 +42,26 @@ function CommunityIndexPage() {
         <Button
           className="block md:inline-block focus:outline-none"
           text="Explore Discussions"
```

Comment:
```suggestion
          text="Explore AsyncAPI Discussions"
```

For accessibility, let's make sure that all CTAs (such as this button text) are as specific as possible so that the users (and screen readers) can clearly tell where this URL goes.